src/schema/v2/artworksForUser/helpers.ts

```diff
@@ -151,20 +151,25 @@ export const getBackfillArtworks = async (
       marketing_collection_id: "top-auction-lots",
     })
 
-    return hits
+    return { artworks: hits, totalCount: hits.length }
   }
 
-  const { body: setsBody } = await setsLoader({
+  const data = await setsLoader({
```

Comment:
**question**: I don't see data referenced anywhere else apart from `const { body: setsBody } = data`, maybe this change is not needed anymore?